src/Language/DifferentialDatalog/Attribute.hs

```diff
@@ -197,6 +203,23 @@ tdefGetSharedRefAttr d tdef =
          Left e  -> error e
          Right b -> b
 
+{- 'alias' attribute: Tells DDlog not to generate Rust declaration for the type
```

Comment:
this may be useful for ddlog programmers too, so perhaps it should be documented.
but if it only applies to scalar types the value could be limited?

Reply:
I haven't thought the implications through, so it's probably better to reserve this for internal use only.  This is only needed for types that conflict with Rust builtins. There is a small and finite number of those.